lib/Chat/Parser/SystemMessage.php

```diff
@@ -403,6 +403,7 @@ public function parseMessage(Message $chatMessage): void {
 					$chatMessage->setMessageType(ChatManager::VERB_MESSAGE);
 				}
 			} catch (\Exception $e) {
+				$chatMessage->setVisibility(false);
```

Comment:
This is too generic. We still want it in the chat. Just not in the right sidebar.

My idea would be to filter them out in `getObjectsSharedInRoomOverview` and `getObjectsSharedInRoom`

We can detect it by checking the system message attribute for `file_shared` and then not having a `file` entry in the message parameter list